### PR TITLE
Explore

### PR DESCRIPTION
**Pull request by m26unkwn**
Added the explore page.
1. Sorting is done
2. Data still left.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,6 +18,20 @@ import {
 } from "./screens";
 
 function App() {
+  const { videoDispatch } = useVideos();
+  let videoConfig = {
+    method: "get",
```
Comment (Debarshi95) [inline src/App.js]: You can move this video outside the component. Putting it inside will create new instance with each render

File: src/components/Cards/video/index.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+import React from "react";
+import "./videoCard.css";
+import { Link } from "react-router-dom";
+import { ReactComponent as MenuIcon } from "../../../assets/ElipsesIcon.svg";
```
Comment (Debarshi95) [inline src/components/Cards/video/index.jsx:6]: Use absolute path. Remove  .. /../

File: src/screens/explore/Explore.jsx [hunk 3]
```diff
@@ -1,7 +1,73 @@
...
+    },
+    {
+      title: "Html",
+      param: "html",
+    },
+    {
+      title: "Css",
+      param: "css",
+    },
+  ];
```
Comment (Debarshi95) [inline src/screens/explore/Explore.jsx]: This too can be moved out side the component

**Review by Debarshi95 — APPROVED** [review]
Loved the way you used custom hook. Some suggestions to work on.

File: src/utils/filters/video-filter.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+      (video) => video.category.toUpperCase() === filter.toUpperCase()
+    );
+  }
+  return videos;
+};
+// filter.length > 1
+//   ? videos.filter(
+//       (video) => video.category.toUpperCase() === filter.toUpperCase()
+//     )
+//   : videos;
```
Comment (saurabhchirde) [inline src/utils/filters/video-filter.js:17]: You can remove these comments if not going to use them.

File: src/App.js (hunk 1 above)
Comment (m26unkwn) [reply src/App.js]: Thanks, @Debarshi95  for suggesting this will implement this.

File: src/components/Cards/video/index.jsx (hunk 2 above)
Comment (m26unkwn) [reply src/components/Cards/video/index.jsx:6]: Don't know about it will learn and then implement it.

File: src/screens/explore/Explore.jsx (hunk 3 above)
Comment (m26unkwn) [reply src/screens/explore/Explore.jsx]: will do it.

**Review by saurabhchirde — APPROVED** [review]
Your cards are pretty good, they are clean and more focused on video which is a good thing.
For mobile responsive just check for the screen size below 400px. It's not accessible. 
Otherwise, looks good to merge.

Screenshot 2022-04-14 165529

File: src/utils/filters/video-filter.js (hunk 4 above)
Comment (m26unkwn) [reply src/utils/filters/video-filter.js:17]: will remove it.

**Pull request merged**